content/docs/typechecking-with-proptypes.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: typechecking-with-proptypes
-title: Typechecking With PropTypes
+title: Checagem de tipos com PropTypes
 permalink: docs/typechecking-with-proptypes.html
 redirect_from:
   - "docs/react-api.html#typechecking-with-proptypes"
 ---
 
 > Note:
+> 
+> `React.PropTypes` foi movido para um package diferente desde a versão 15.5 do React. Por favor, use [a biblioteca `prop-types`](https://www.npmjs.com/package/prop-types).
```

Comment:
```suggestion
> `React.PropTypes` foi movido para um pacote diferente desde a versão 15.5 do React. Por favor, use [a biblioteca `prop-types`](https://www.npmjs.com/package/prop-types).
```

As suggested by the Style Guide https://github.com/reactjs/pt-BR.reactjs.org/issues/4